packages/cli/src/commands/composite/create.ts

```diff
@@ -23,12 +23,21 @@ export default class CreateComposite extends Command<Flags, { schemaFilePath: st
       char: 'o',
       description: 'path to the file where the composite representation should be saved',
     }),
+    index: Flags.boolean({
+      char: 'i',
+      description: 'If indexing should occur when creating composites (default=true)',
```

Comment:
Should the flag maybe be called "deploy" instead of "index"?  We should try to be consistent with whatever other naming conventions we are using elsewhere